src/SFA.DAS.ReferenceData.Api/Controllers/OrganisationController.cs

```diff
@@ -109,6 +110,18 @@ public async Task<IHttpActionResult> SearchOrganisations(string searchTerm = "",
             return Ok(response.Organisations);
         }
 
+        [Route("getLocateableOrganisationTypes")]
+        [HttpGet]
+        [ApiAuthorize]
+        public async Task<IHttpActionResult> GetLocateableOrganisationTypes()
```

Comment:
I'm not sure about the naming of this method, what do you think to something like `GetIdentifiableOrganisationTypes`?